README.md

```diff
@@ -1,208 +1,262 @@
-<img src="assets/images/apn-step4-1.PNG"/>
+### Step 9 - Build the hybrid mobile application
 
-### Step 5 - Generating Certificate 
-Click continue
-<img src="assets/images/apn-step5.PNG"/>
+You will need to build the hybrid mobile application. Refer to the [Publishing a Mendix Hybrid Mobile App howto] (https://world.mendix.com/display/howto50/Publishing+a+Mendix+Hybrid+Mobile+App+in+Mobile+App+Stores) to get the explanation on how to do that. Note that you should opt to download the app instead of directly publishing it. This is necessary because we need to include a phonegap plugin which is used by this module into the mobile hybrid application.
 
-You will then be asked to upload a certificate signing request file. In order to create this file please read the following documentation:
+Once you have the mobile hybrid project file downloaded, extract it and include the required phonegap plugin by adding this line to the `config.xml` file:
 
-[Creating CSR](https://world.mendix.com/display/refguide5/Managing+App+Signing+Keys)
+```
+<gap:plugin name="com.phonegap.plugins.pushplugin" version="2.5.0" />
+```
 
-Upload the CSR and then click generate. You will be presented by a screen saying your certificate is ready.
-Click done and then click on your certificate from the list and click Download.
+You can proceed by repackaging the project into a zip file and use PhoneGap Build to generate the files for Android and iOS.
 
-### Step 6 - Converting Certificate
-Now that we have the certificate from apple we now need to convert this into p12 format so that we can get it to work with our Mendix application.
-This documentation should be helpful for getting your certificate converted:
+For more information about PhoneGap Build, you can refere to their [documentation](http://docs.build.phonegap.com/en_US/index.html).
 
-[Converting Cer to p12](http://docs.build.phonegap.com/en_US/signing_signing-ios.md.html)
+## Testing The Implementation
 
-### Step 7 - Setting up Mendix APNS
-Once you have the p12 certificate you can set up the apple push notification system in Mendix. Login as an admin to the application and open up the apple admin. In the configuration you will need to upload the p12 file to the apple administration section. You will also need to include the passcode that you entered when you converted the file to a p12 format.
+Once you finished implementing the steps described previously, you might want to test whether everything is already in place. This can be done easily by making use of the administration pages that should have been included into your application. Follow these steps to send a push notification to a device:
```

Comment:
'you will want to test' (to make it an even stronger suggestion than might?)

Suggestion: 'This can be done easily using the administration pages that should have been included *in* your application during step 7 of the implementation guide' (ideally probably with link to that step)